packages/flutter_tools/lib/src/macos/xcode.dart

```diff
@@ -17,6 +17,25 @@ const int kXcodeRequiredVersionMinor = 2;
 
 Xcode get xcode => context.get<Xcode>();
 
+enum SdkType {
+  iPhone,
+  iPhoneSimulator,
+  macOS,
+}
+
+String getNameForSdk(SdkType sdk) {
```

Comment:
dartdoc these (i.e. where is the naming source of truth)

Reply:
Totally undocumented by Apple AFAIK.  When people Google it they will find my Flutter comment!